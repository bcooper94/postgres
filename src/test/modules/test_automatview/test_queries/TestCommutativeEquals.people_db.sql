SET automatview.training_sample_count = 3;

SELECT name, age FROM person JOIN employee ON person_id = person.id JOIN manager ON employee.id = employee_id
    JOIN executive ON manager_id = manager.id;

SELECT name, age FROM person JOIN employee ON person.id = person_id JOIN manager ON employee.id = employee_id
    JOIN executive ON manager_id = manager.id;

SELECT name, age FROM person JOIN employee ON person.id = person_id JOIN manager ON employee_id = employee.id
    JOIN executive ON manager_id = manager.id;

/* Matview query */
SELECT name, age FROM person JOIN employee ON person_id = person.id JOIN manager ON employee_id = employee.id
    JOIN executive ON manager.id = manager_id;