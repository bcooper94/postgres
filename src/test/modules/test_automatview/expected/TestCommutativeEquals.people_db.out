1
SELECT Matview.person_name AS Matview_person_name, Matview.person_age AS Matview_person_age FROM Matview